odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -199,6 +201,98 @@ def window_fcn(x):
     return ray_trafo.range.element(window_fcn) / n_half_rot
 
 
+def parker_weighting(ray_trafo, q=0.25):
+    """Create parker weighting for a `RayTransform`.
+
+    The Parker weigtings are weighting functions that ensure that oversampled
+    counted twice in reconstructions, with partial angular intervals. It is
+    useful in analytic reconstruction methods such as FBP to give a more
+    accurate reconstruction and can improve convergence rates for iterative
+    methods.
+
+    See WES2002_ for more information.
```

Comment:
I don't quite like this citation style, the link text doesn't really make sense. Either use proper citation style or use a better link text.